experimental/packages/opentelemetry-sdk-metrics-base/test/ExemplarFilter.test.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import { ROOT_CONTEXT, SpanContext, TraceFlags, trace } from '@opentelemetry/api';
+
+import {
+  AlwaysSampleExemplarFilter,
+  NeverSampleExemplarFilter,
+  WithTraceExemplarFilter
+} from '../src/exemplar/';
+
+describe('ExemplarFilter', () => {
+  const TRACE_ID = 'd4cda95b652f4a1592b449d5929fda1b';
+  const SPAN_ID = '6e0c63257de34c92';
+
+	describe('AlwaysSampleExemplarFilter', () => {
```

Comment:
Please indent with 2 white spaces.

Reply:
I am not sure why this is showing up like this. It is indent with 2 white spaces only.